### ISSUE TITLE
Fix outstanding issues with interactive evolution interface

### ISSUE THREAD

**Issue by schrum2** [schrum2/MM-NEATv2#451]
1. In simplified interfaces, disable the Animation Length slider entirely.
2. Make the default "Pause between frames" slightly longer. In particular, 3D objects should rotate slower, but all animations should be slower
3. In simplified interface for Breedesizer, remove the clip length slider
4. Go ahead and make the "Color of Objects" drop down hidden in the simplified interface (just evolve the colors)

Basically, we want to make the simplified interface easy to use and not confusing. We don't want different users to have strongly different experiences depending on how much they explore the interface. We also want to avoid having a lot of settings that cause extra loading times for animations, since the loading times are annoying.
